server/lib/nl/detection/types.py

```diff
@@ -346,6 +346,27 @@ class NLClassifier:
   attributes: List[ClassificationAttributes]
 
 
+class ActualDetectorType(str, Enum):
+  """Enum to represent detector types"""
+  Heuristic = "Heuristic Based"
+  LLM = "LLM Based"
+  # No fallback
```

Comment:
nit: understood what this is trying to do but I wonder if there should be two enums: one which decides whether it's a Hybrid or Not and the other which has the actual detector type used (LLM, Heuristic, LLMPlace, LLMVar).

Reply:
This enum is basically just a stand-in for the returned string, so maybe 1 enum is simpler overall?